docs/sql/joins.txt

```diff
@@ -108,18 +108,16 @@ each shard of the used tables.
 Limitations
 ...........
 
- - Joining more than 2 tables is currently not possible
+ - Joining more than 2 tables can result in execution plans which perform
+   poorly as there is no query optimizer in place yet.
 
+ - The ORDER BY clause must not contain conditions if more than 2 tables are
```

Comment:
at least I don't really understand this limitation from your description here ;) (as already discussed)